File: CHW058, 2014.06.12/Swift/CL/main.c
// clang main.c -Ofast -o cmain

#include <stdio.h>

int main(int argc, char** argv)
{
    double array[1000000];
    
    for(size_t i = 0; i < 1000000; ++i) {
        array[i] = (double)i * (double)i;
    }
    
    return 0;
}

File: CHW058, 2014.06.12/Swift/CL/main.m
// clang main.m -O3 -framework Foundation -o objcmain

#import<Foundation/Foundation.h>

int main(int argc, char** argv)
{
    NSMutableArray *array = [NSMutableArray array];
    
    for(NSInteger i = 0; i < 1000000; ++i) {
        array[i] = @((double)i * (double)i);
    }
    
    NSLog(@"%lu iterations", (unsigned long)array.count);
        
    return 0;
}
